CoreScriptsRoot/StarterScript.lua

```diff
@@ -57,3 +57,4 @@ end
 if useVRKeyboard then
 	require(RobloxGui.Modules.VR.VirtualKeyboard)
 end
+
```

Comment:
Gotta have dat new line at the end of a file man, you don't want to surprise the compiler with an EOF marker out of nowhere!